website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js

```diff
@@ -51,4 +51,9 @@ app.controller('MainController',
         var cmdlet = productInfo[$routeParams.product][$routeParams.cmdlet];
         return cmdlet.parameters;
       };
+      
+      /* Tells us if the object passed in is empty */
```

Comment:
Prefer single line comments, except when providing type annotations.

Using multi-line comments makes it harder to comment-out large blocks of code. (Since it would terminate early.)